aiida/orm/computers.py

```diff
@@ -131,6 +133,18 @@ def full_text_info(self):
         else:
             ret_lines.append('   # No append text.')
 
+        ret_lines += [' * Configured users:']
+        all_users = users.User.objects(self.backend).all()
```

Comment:
No need to explicitly pass `self.backend`

Reply:
ok